tests/unit/lms/services/rsa_key_service_test.py

```diff
@@ -65,6 +65,11 @@ def test_get_all_public_jwks(self, svc, valid_keys):
         ]
         assert keys == Any.list.containing(expected_keys).only()
 
+    def test_get_random_key(self, svc, valid_keys):
+        key = svc.get_random_key()
```

Comment:
I'm not testing the randomness element of this

Reply:
Yeah, kind of logically impossible to do really. You could call it 1 million times and technically get the same one everytime.